test/libsolidity/syntaxTests/parsing/mapping_with_names_5.sol

```diff
@@ -0,0 +1,4 @@
+contract test {
+    mapping(address owner => mapping(address spender => bytes32[] notes)) names;
```

Comment:
Can you add a test where these symbols conflict with an existing definition? As an example add a `address owner` as a state variable.